tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsUnitTest.java

```diff
@@ -769,22 +762,12 @@ public void testPropagateInvalidJsonError() {
     }
 
     @SuppressWarnings("unchecked")
-    private static <T> scala.collection.immutable.Set<T> set(final T... set) {
-        return mutableSet(set).toSet();
-    }
-
-    @SuppressWarnings({"deprecation", "unchecked"})
-    private static <T> scala.collection.mutable.Set<T> mutableSet(final T...set) {
-        return JavaConverters.asScalaSet(new HashSet<>(Arrays.asList(set)));
+    private static <T> Set<T> set(final T... set) {
```

Comment:
Were we also not unable to remove these warnings?

Reply:
Fixed. Yes, we able to remove these method.

Reply:
Can we remove this as well?

Reply:
Done